fabtests/common/shared.c

```diff
@@ -1656,7 +1656,7 @@ int ft_read_addr_opts(char **node, char **service, struct fi_info *hints,
 {
 	int ret;
 
-	if (opts->dst_addr && (opts->src_addr || !opts->oob_port)){
+	if (opts->dst_addr) {
```

Comment:
@rwespetal - I believe this change revert is resulting in breaking EFA testing.  I need to understand what EFA is needing, so the correct update can be added.

Reply:
We're using the -E option for out of band setup. Unfortunately in band communication of the addresses doesn't work with EFA + fabtests at the moment.

Reply:
The -E option makes sense.  The issue I'm seeing is that the original code would call fi_getinfo with FI_SOURCE set, but set node and service to NULL on input.  The test case is calling a fabtest (fi_unexpected_msg) as a client, with only the destination address specified.  That test uses an OOB sync mechanism.

The tests use of an OOB connection should be independent of how the test calls fi_getinfo.  But in the existing code, it is not.